scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -37,27 +36,26 @@ def _build_arg_parser():
                         'supported format.\nCan be a binary mask or a ' +
```

Comment:
Should we use a capital letter at the beginning of each help/sentence?
I think this is the case everywhere